### DISCUSSION TITLE
1 Archive multiple Plotter

### DISCUSSION THREAD

**Discussion by socramm9** [ericaltendorf/plotman#440] [Q&A] [answered]
I am trying to configure archiving for my 2 Plotter pc.

Looking in https://github.com/ericaltendorf/plotman/wiki/Archiving it seams that it is not possible to configure more then 1 allowed IP

etc/rsyncd.conf
```
...
# since we dont use auth only accept connections from plotter's ip    
hosts allow = plotter.ip.address
```

Is it maybe posible to configure more then 1 address? Or just to allow all? I have an extra Firewall for my plotter network so I think it should be not a problem...?

**Comment by thomann061** [comment, marked as answer]
`hosts allow = 192.168.1.101, 192.168.1.102`

**Comment by randellhodges** [comment]
You can also specify an entire subnet. `192.168.1.0/24`
